### PR TITLE
Adds default tags for the apache-debian and fpm

### PR DESCRIPTION
**Pull request by tobybatch**
So you can now just use kimai/kimai2:fpm-alpine and always get the latest image.

### CODE REVIEW

File: bin/release.sh [hunk 1]
```diff
@@ -31,8 +31,10 @@ $BIN_DIR/simple-test.sh $TAG
 
 set -x
 docker tag kimai/kimai2:fpm-alpine-${TAG}-prod kimai/kimai2:latest
+docker tag kimai/kimai2:fpm-alpine-${TAG}-prod kimai/kimai2:fpm-alpine
 docker tag kimai/kimai2:fpm-alpine-${TAG}-prod kimai/kimai2:latest-prod
 docker tag kimai/kimai2:apache-debian-${TAG}-dev kimai/kimai2:latest-dev
+docker tag kimai/kimai2:apache-debian-${TAG}-dev kimai/kimai2:apache-debian
```
Comment (kevinpapst) [inline bin/release.sh:37]: are there prod images for apache or is apache always a dev version with sqlite? 
probably that tag name could be misunderstood?

File: bin/release.sh [hunk 2]
```diff
@@ -31,8 +31,10 @@ $BIN_DIR/simple-test.sh $TAG
 
 set -x
 docker tag kimai/kimai2:fpm-alpine-${TAG}-prod kimai/kimai2:latest
```
Comment (kevinpapst) [inline bin/release.sh:33]: One question, outside that PR - but I stumbled upon it while reading through the docs right now... 
The doc at https://tobybatch.github.io/kimai2/dev-instance.html#dev-instances say that `kimai/kimai2:latest` is a apache-dev instance. 
Are the docs wrong here?

File: bin/release.sh (hunk 1 above)
Comment (kevinpapst) [reply bin/release.sh:37]: And btw: is there a docu somewhere explaining the different tags?

**Review by jokay — APPROVED** [review]
Good idea 👍🏼

**Comment by tobybatch** [comment]
Kevin is of course right, damn his German (Austrian?) efficiency. :+1: 

I'll review the docs and add explanations of the different tags and resubmit the PR.

**Pull request closed without merging**